spec/library_spec.rb

```diff
@@ -0,0 +1,23 @@
+require './lib/library.rb'
+require 'date'
+
+describe Library do
+books = [{ title: 'Alfons leker', author: 'A. Andersson' },{ title: 'Bible', author: 'TGAOTU' } ]
```

Comment:
Metrics/LineLength: Line is too long. [97/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)